applications/sandbox/engine/shaders/bloomcombine.shs

```diff
@@ -1,10 +1,11 @@
 #version 450
 #state CULL OFF
 #state DEPTH OFF
-generate(vertex, fragment)
 #state ALPHA OFF
-#include <stdio.shinc>
 
+generate(vertex, fragment)
+
+#include <stdio.shinc>
```

Comment:
this is unrelated to the review, but is this done for _aesthetic_ purposes or is this an issue with the preprocessor ?

Reply:
_aesthetic_